docs/_documentations/gettingstarted-overview.md

```diff
@@ -65,13 +75,6 @@ To get started with using Codewind, choose from one of the following configurati
 
 **To use Codewind for Eclipse Che** select [Using Codewind as a Hosted Application in your Cloud](./eclipseche-codewind-overview.html)
 
-**To use Codewind remotely** you must first install Codewind locally by selecting from:
-
-- [Installing Codewind for VS Code](./vsc-getting-started.html)
-- [Installing Codewind for Eclipse](./eclipse-getting-started.html)
-- [Installing Codewind for intellij](./intellij-getting-started.html)
-
-Then, choose from the following options:
-  1. Deploy Codewind to your cloud if not already done so by you or a sysadmin or a DevOps engineer by following instructions to [Deploy Codewind Remotely](./remote-deploying-codewind.html).
-  2. Connect your Codewind extension of your local desktop IDE to Codewind in your cloud by following instructions for [Using Codewind Remotely](./remote-codewind-overview.html). 
-
+<div class="callout">
+	<b>Note:</b> To use Codewind remotely, you must first install Codewind locally. For more information, see instructions for <a href="./remote-deploying-codewind.html"> deploying Codewind Remotely</a> . After Codewind is deployed remotely, you can connect your Codewind extension of your local desktop IDE to Codewind in your cloud by following instructions for <a href="./remote-codewind-overview.html">using Codewind remotely.</a>
```

Comment:
First link in the paragraph has a space after the </a> before the full stop, so remove space.  

Last link in this paragraph, the period is inside the link text, would be better outside, for example: 
<a href="./remote-codewind-overview.html">using Codewind remotely</a>.